articles/container-instances/container-instances-region-availability.md

```diff
@@ -27,6 +27,7 @@ The following regions and maximum resources are available to container groups wi
 | Region | Max CPU | Max Memory (GB) | VNET Max CPU | VNET Max Memory (GB) | Storage (GB) | GPU SKUs (preview) |
 | -------- | :---: | :---: | :----: | :-----: | :-------: | :----: |
 | Australia East | 4 | 16 | 4 | 16 | 50 | N/A |
+| Australia Southeast | 4 | 14 | 4 | 16 | 50 | N/A |
```

Comment:
VNet is not supported in this region yet.
```suggestion
| Australia Southeast | 4 | 14 | N/A | N/A | 50 | N/A |
```

Reply:
Good pickup MacKenzie!